openedxstats/apps/sites/templates/sites/ot_chart.html

```diff
@@ -88,12 +89,11 @@
                 dataType: 'json',
                 type: 'POST',
                 success: function (data) {
-                    snapshotList = JSON.parse(data);
+                    snapshotList = data;
```

Comment:
Maybe get rid of this assignment, it was to make testing the skipped parsing easier.